### PR TITLE
fix acpacity expansion IT

### PR DESCRIPTION
**Pull request by RemHero**

### CODE REVIEW

File: test/src/test/java/cn/edu/tsinghua/iginx/integration/expansion/IoTDBHistoryDataCapacityExpansionIT.java [hunk 1]
```diff
@@ -183,7 +185,7 @@ public void testQueryAfterInsertNewDataFromNoInitialNode() throws Exception {
 
     //@Test
     public void testCapacityExpansion_oriHasDataExpNoData() throws Exception {
-        session.executeSql("ADD STORAGEENGINE (\"127.0.0.1\", 6668, \"" + ENGINE_TYPE + "\", \"username:root, password:root, sessionPoolSize:20, has_data:true, is_read_only:false\");");
+        session.executeSql("ADD STORAGEENGINE (\"127.0.0.1\", 6668, \"" + ENGINE_TYPE + "\", \"username:root, password:root, sessionPoolSize:20, has_data:no, is_read_only:true\");");
```
Comment (zhuyuqing) [inline test/src/test/java/cn/edu/tsinghua/iginx/integration/expansion/IoTDBHistoryDataCapacityExpansionIT.java:188]: Are you sure here "hasdata:no" is correct?

File: test/src/test/java/cn/edu/tsinghua/iginx/integration/scaleout/IoTDBRestfulScaleOutIT.java [hunk 2]
```diff
@@ -1,34 +1,77 @@
 package cn.edu.tsinghua.iginx.integration.scaleout;
```
Comment (zhuyuqing) [inline test/src/test/java/cn/edu/tsinghua/iginx/integration/scaleout/IoTDBRestfulScaleOutIT.java:1]: IoTDBRestfulScaleOutIT seems to have so similar functionality as IoTDBSqlScaleOutIt and IoTDBTagScaleOutIT. But why do they differ so much in their respective testing process and implementation?
Why not make a better abstraction for them to clear up the code?

File: .github/workflows/scale-out.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@ concurrency:
 
 jobs:
   IoTDB-Tag-Test:
```
Comment (zhuyuqing) [inline .github/workflows/scale-out.yml:17]: Tag-Test seems to have similar testing objectives as session test ad rest test in the same file. So why not structure them similarly?

File: test/src/test/java/cn/edu/tsinghua/iginx/integration/rest/RestIT.java [hunk 4]
```diff
@@ -175,7 +148,11 @@ public void capacityExpansion() throws Exception {
     }
 
     @Test
-    public void testQueryWithoutTags(){
+    public void testQueryWithoutTags() throws Exception {
+        SessionExecuteSqlResult res = session.executeSql("show time series");
```
Comment (zhuyuqing) [inline test/src/test/java/cn/edu/tsinghua/iginx/integration/rest/RestIT.java:152]: Can these session commands be implemented as REST commands?

**Review by zhuyuqing — CHANGES_REQUESTED** [review]
@RemHero Would you please update the structure of scaleout tests to let similar tests have similar structure? Thanks.

**Comment by SolomonAnn** [comment]
Remove useless files, including `.github/actions/iginxRunner/iginxLog.txt`, `.github/actions/iotdbRunner/iotdb1Log.txt` and `.github/actions/iotdbRunner/iotdb2Log.txt`.

File: .github/actions/iginxRunner/action.yml [hunk 5]
```diff
@@ -16,8 +16,16 @@ runs:
...
-        chmod +x "${GITHUB_WORKSPACE}/core/target/iginx-core-${{inputs.version}}/sbin/start_iginx.sh"
-        nohup "${GITHUB_WORKSPACE}/core/target/iginx-core-${{inputs.version}}/sbin/start_iginx.sh" &
+        if [ "$RUNNER_OS" == "Linux" ]; then
+          chmod +x "${GITHUB_WORKSPACE}/.github/iginxRunnerLinux.sh"
+          "${GITHUB_WORKSPACE}/.github/iginxRunnerLinux.sh"
+        elif [ "$RUNNER_OS" == "macOS" ]; then
+          chmod +x "${GITHUB_WORKSPACE}/.github/iginxRunnerMacOS.sh"
+          "${GITHUB_WORKSPACE}/.github/iginxRunnerMacOS.sh"
+        fi
+#        chmod +x "${GITHUB_WORKSPACE}/core/target/iginx-core-${{inputs.version}}/sbin/start_iginx.sh"
```
Comment (SolomonAnn) [inline .github/actions/iginxRunner/action.yml:26]: Delete unused lines.

**Review by SolomonAnn — APPROVED** [review]

**Review by zhuyuqing — APPROVED** [review]

**Pull request merged**